.github/workflows/github-ci.yml

```diff
@@ -22,20 +22,20 @@ jobs:
     pipeline:
         name: jhipster-online pipeline
         runs-on: ubuntu-latest
-        if: "!contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.pull_request.title, '[skip ci]') && !contains(github.pull_request.title, '[ci skip]')"
+        if: "!contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.pull_request.title, '[skip ci]') && !contains(github.event.pull_request.title, '[ci skip]')"
         timeout-minutes: 40
         env:
-            NODE_VERSION: 10.16.3
+            NODE_VERSION: 12.16.0
```

Comment:
you need to be consistent with https://github.com/jhipster/jhipster-online/blob/master/pom.xml#L20 too

Then, we should migrate to Node 12.16.1 for all our projects... I didn't have time to do it yet